src/index.scss

```diff
@@ -6,18 +6,32 @@
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
 }
-
+*{
+  margin: 0px;
+  padding: 0px;
+  border:none;
+}
+*,
+*::before,
+*::after{
+  box-sizing: border-box;
+}
 
 body {
-  margin: 0;
-  display: flex;
-  place-items: center;
+  
+  // display: flex;
```

Comment:
the best practice is not to have commented lines in code

Reply:
Fixed.